app/views/user_email_mailer/response_invite.html.erb

```diff
@@ -3,215 +3,139 @@
 <head>
   <title>Email letter</title>
   <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
   <style>
-      @import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@600;700&display=swap');
       * {
           margin: 0;
           padding: 0;
           box-sizing: border-box;
-          font-family: 'Josefin Sans',  sans-serif;
+          font-family: 'Josefin Sans', 'Arial', 'Ubuntu',  sans-serif;
           font-style: normal;
+          font-weight:700;
+          text-align: center;
       }
-      .width-screen{
+      .board{
           min-width:1060px;
       }
-      .convert {
-          width: 127px;
-          height: 58px;
-          text-align: center;
+      .centre {
           margin: 0 auto;
-          background: #D9D9D9;
-          border: 1px solid #000000;
       }
-      .convert p {
+      @media only screen and (min-width: 1080px){
+          .emotions{
+            width: 1060px;
+          }
+      }
+      .flex{
+          display: flex;
```

Comment:
whether we decided to follow bootstrap stylings, so we can use bootstrap class naming. For example here - `.d-flex`.  Rename it pls